softbinding-algorithm-entry-schema.json

```diff
@@ -8,6 +8,11 @@
             "maximum": 65535,
             "description": "This identifier will be assigned when the soft binding algorithm is added to the list."
         },
+        "deprecated": {
+            "type": "boolean",
+            "default": false,
+            "description": "Indicates whether this soft binding algorithm is deprecated. Deprecated algorithms shall not be used for creating soft bindings.  Deprecated algorithms may be used for resolving soft bindings but this behaviour is discouraged."
```

Comment:
@domguinard, can you please provide details by responding to this comment as to why this behavior is discouraged. Thank you!

Reply:
Hi @BadrTad - what is discouraged is the use of deprecated algorithms to recover missing manifests. The idea is simply that if an algorithm is deprecated there must be a reason (e.g., a new version, a known issue, etc.) and hence it is better to move to different algorithms. Having said that, if an asset embeds a deprecated watermark for instance, a client might still want to use it to recover a manifest.
Note that this text and the deprecated field have already been accepted and merged into the core spec, what was missing was to update the copy of the schema in this repo with this additional field.